src/Compilers/Core/CodeAnalysisTest/FileSystem/PathUtilitiesTests.cs

```diff
@@ -383,5 +383,13 @@ public void CombinePaths_DifferentFromPathCombine(string expected, string path1,
         {
             Assert.Equal(expected, PathUtilities.CombinePaths(path1, path2));
         }
+
+        [ConditionalFact(typeof(WindowsOnly)), WorkItem(51602, @"https://github.com/dotnet/roslyn/issues/51602")]
```

Comment:
Why WindowsOnly?

Reply:
`PathUtilities.GetRelativePath(@"C:\A\B\", @"C:\A\B")` is handled completely different by the underlying framework on *nix machines

Reply:
Is there any analogous bug that we should verify is fixed on non-Windows platforms?

Reply:
I'll add a test